### ISSUE TITLE
Expose original ga object

### ISSUE THREAD

**Issue by yangshun** [react-ga/react-ga#64]
At the current state, `react-ga` does not cover everything that `analytics.js` has to offer. It is also hard for this library to continuously be up to date with the developments of the Google Analytics library. 

Hence it will be nice if the library exposes the original Google Analytics object so that users can directly use the original `ga` object for features that `react-ga` did not implement.

**Comment by karanjthakkar** [comment]
@yangshun Are you thinking about adding support for this anytime soon? I was thinking of adding a method, like for eg: `send`, which would allow people to send any kind of analytics (`hitType`) directly without being normalized in some way by the library.

eg:
```
/*
* Example of fielObject
* fieldObject = {
*     hitType: 'pagview',
*     page: 'Test'
* }
*
*/
send(fieldObject) {
    ga(send, fieldObject);
}
```

Thoughts?

**Comment by yangshun** [comment]
Hi @karanjthakkar , #65 would be more relevant to what you were suggesting. PRs are welcome (:

**Comment by karanjthakkar** [comment]
@yangshun Fantastic! I'll send across a PR in some time and we can get this merged right away. We are using this in a project.

**Comment by yangshun** [comment]
Awesome, looking forward to it 😄

**Comment by yangshun** [comment]
Fixed via f26a385
